website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -661,7 +661,8 @@ Properties without a default value are required.
 | `data.attributes.vcs-repo.identifier`         | string         | (previous value) | A reference to your VCS repository in the format :org/:repo where :org and :repo refer to the organization and repository in your VCS provider. The format for Azure DevOps is `:org/:project/_git/:repo`.                                                                                                                                                                                                         |
 | `data.attributes.vcs-repo.ingress-submodules` | boolean        | (previous value) | Whether submodules should be fetched when cloning the VCS repository.                                                                                                                                                                                                                                                                                                                                              |
-| `data.attributes.vcs-repo.oauth-token-id`     | string         | (previous value) | The VCS Connection (OAuth Connection + Token) to use. This ID can be obtained from the [oauth-tokens](/terraform/cloud-docs/api-docs/oauth-tokens) endpoint.                                                                                                                                                                                                                                                                 |
+| `data.attributes.vcs-repo.oauth-token-id`     | string         | (previous value) | The VCS Connection (OAuth Connection + Token) to use. This ID can be obtained from the [oauth-tokens](/terraform/cloud-docs/api-docs/oauth-tokens) endpoint. This value must not be specified if `github-app-installation-id` is specified.                                                                                                                                                                                                                                                               |
+| `data.attributes.vcs-repo.github-app-installation-id` | string | (previous value) | The VCS Connection GitHub App Installation to use. This ID can be obtained from the user settings page. Requires previously authorizing the GitHub App and generating a user-to-server token. You can manage this from User Settings within Terraform Cloud. This value must not be specified if `oauth-token-id` is specified.                                                                                                                                             |
```

Comment:
I think this text is copy of what was on the modules page? If so please use suggestions from there if they're correct 🙏🏻

Reply:
Yes you are correct. Will apply feedback from the modules page to here as well